Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -173,10 +173,10 @@ public final class PostgresJobQueue: JobQueueDriver {
     /// Push Job onto queue
     /// - Returns: Identifier of queued job
     @discardableResult public func push<Parameters>(_ jobRequest: JobRequest<Parameters>, options: JobOptions) async throws -> JobID {
-        let buffer = try self.jobRegistry.encode(jobRequest: jobRequest)
+        //.let buffer = try self.jobRegistry.encode(jobRequest: jobRequest)
```

Comment:
Is this left as a comment intentionally?

Reply:
Most likely not. I'll fix it later today